packages/sfui/frameworks/nuxt/playground/nuxt.config.ts

```diff
@@ -0,0 +1,26 @@
+export default defineNuxtConfig({
+  modules: ['../src/module.ts'],
+  storefrontUi: {
+    contentPath: './../../../../node_modules/@storefront-ui/vue/dist/**/*.mjs',
+  },
+  imports: {
+    transform: {
+      // you could also add the path of your built library to prevent this happening
+      // for your users, but the issue is probably only replicable in your monorepo
+      exclude: [/.*dist\/index\.mjs/],
+    },
+  },
+  tailwindcss: {
+    config: {
+      theme: {
+        extend: {
+          colors: {
+            primary: {
+              700: '#1e1e1e',
+            },
+          },
+        },
+      },
+    },
+  },
+} as any);
```

Comment:
Why `as any` is needed?

Reply:
Fix required bumping Nuxt version to `3.3` - updated preview app as well.